### ISSUE TITLE
Compatibility with typo3 7.6.15

### ISSUE THREAD

**Issue by MrGecco** [networkteam/nwt_imagecrop#1]
Can anyone confirm that the extension is working with typo3 7.6.15? Put quite some effort in it but couldn't get it up and running.

The installation process proceeds without any errors, the static template "Image Cropscaling (nwt_imagecrop)" is included in the template and the additional database columns are created in the table tt_content. However, the extension doesn't appear in the content elements tab "Appearance".

**Comment by christophlehmann** [comment]
Hi,

I checked it in 7.6.16 succesfully. The fields appear in an image element.

Please check the configuration backend module => TCA => tt_content => palettes => image_settings => show_item

When it doesn't contain the new fields you probably need to clear the system cache

**Comment by MrGecco** [comment]
Hey,

thank you very much for your answer. I checked it again in version 7.6.16 and the "configuration backend module => TCA => tt_content => palettes => image_settings => show_item" is empty. 

I enabled cleaning the system cache for my user via `options.clearCache.system = 1` in TSConfig. Unfortunately, it didn't change anything. I'm really wondering what could be wrong.

I wish you a nice evening and thank you for your effort and your great extensions.
